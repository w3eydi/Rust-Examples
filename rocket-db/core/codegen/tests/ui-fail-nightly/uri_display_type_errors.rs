../ui-fail/uri_display_type_errors.rs