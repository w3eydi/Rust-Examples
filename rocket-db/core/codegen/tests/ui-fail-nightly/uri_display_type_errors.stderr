error[E0277]: the trait bound `BadType: UriDisplay<rocket::http::uri::fmt::Query>` is not satisfied
 --> $DIR/uri_display_type_errors.rs:6:13
  |
6 | struct Bar1(BadType);
  |             ^^^^^^^ the trait `UriDisplay<rocket::http::uri::fmt::Query>` is not implemented for `BadType`
  |
  = note: required because of the requirements on the impl of `UriDisplay<rocket::http::uri::fmt::Query>` for `&BadType`

error[E0277]: the trait bound `BadType: UriDisplay<rocket::http::uri::fmt::Query>` is not satisfied
  --> $DIR/uri_display_type_errors.rs:10:5
   |
10 |     field: BadType,
   |     ^^^^^^^^^^^^^^ the trait `UriDisplay<rocket::http::uri::fmt::Query>` is not implemented for `BadType`
   |
   = note: required because of the requirements on the impl of `UriDisplay<rocket::http::uri::fmt::Query>` for `&BadType`

error[E0277]: the trait bound `BadType: UriDisplay<rocket::http::uri::fmt::Query>` is not satisfied
  --> $DIR/uri_display_type_errors.rs:16:5
   |
16 |     bad: BadType,
   |     ^^^^^^^^^^^^ the trait `UriDisplay<rocket::http::uri::fmt::Query>` is not implemented for `BadType`
   |
   = note: required because of the requirements on the impl of `UriDisplay<rocket::http::uri::fmt::Query>` for `&BadType`

error[E0277]: the trait bound `BadType: UriDisplay<rocket::http::uri::fmt::Query>` is not satisfied
  --> $DIR/uri_display_type_errors.rs:21:11
   |
21 |     Inner(BadType),
   |           ^^^^^^^ the trait `UriDisplay<rocket::http::uri::fmt::Query>` is not implemented for `BadType`
   |
   = note: required because of the requirements on the impl of `UriDisplay<rocket::http::uri::fmt::Query>` for `&BadType`
   = note: 1 redundant requirements hidden
   = note: required because of the requirements on the impl of `UriDisplay<rocket::http::uri::fmt::Query>` for `&&BadType`

error[E0277]: the trait bound `BadType: UriDisplay<rocket::http::uri::fmt::Query>` is not satisfied
  --> $DIR/uri_display_type_errors.rs:27:9
   |
27 |         field: BadType,
   |         ^^^^^^^^^^^^^^ the trait `UriDisplay<rocket::http::uri::fmt::Query>` is not implemented for `BadType`
   |
   = note: required because of the requirements on the impl of `UriDisplay<rocket::http::uri::fmt::Query>` for `&BadType`
   = note: 1 redundant requirements hidden
   = note: required because of the requirements on the impl of `UriDisplay<rocket::http::uri::fmt::Query>` for `&&BadType`

error[E0277]: the trait bound `BadType: UriDisplay<rocket::http::uri::fmt::Query>` is not satisfied
  --> $DIR/uri_display_type_errors.rs:35:9
   |
35 |         other: BadType,
   |         ^^^^^^^^^^^^^^ the trait `UriDisplay<rocket::http::uri::fmt::Query>` is not implemented for `BadType`
   |
   = note: required because of the requirements on the impl of `UriDisplay<rocket::http::uri::fmt::Query>` for `&BadType`
   = note: 1 redundant requirements hidden
   = note: required because of the requirements on the impl of `UriDisplay<rocket::http::uri::fmt::Query>` for `&&BadType`

error[E0277]: the trait bound `BadType: UriDisplay<rocket::http::uri::fmt::Path>` is not satisfied
  --> $DIR/uri_display_type_errors.rs:40:12
   |
40 | struct Baz(BadType);
   |            ^^^^^^^ the trait `UriDisplay<rocket::http::uri::fmt::Path>` is not implemented for `BadType`
   |
   = note: required because of the requirements on the impl of `UriDisplay<rocket::http::uri::fmt::Path>` for `&BadType`
